internal/plugintypes/timestamp.go

```diff
@@ -12,13 +12,29 @@ type TimestampedBig struct {
 	Value     cciptypes.BigInt `json:"value"`
 }
 
+// TimestampedUnixBig Maps to on-chain struct
+// https://github.com/smartcontractkit/chainlink/blob/37f3132362ec90b0b1c12fb1b69b9c16c46b399d/contracts/src/v0.8/ccip/libraries/Internal.sol#L43-L47
+//
+//nolint:lll //url
+type TimestampedUnixBig struct {
+	Timestamp uint32           `json:"timestamp"`
+	Value     cciptypes.BigInt `json:"value"`
```

Comment:
you might want to use `*big.Int`. I had `deserialization` issue using the chain reader.

Reply:
I think I've been getting some Nil values as well, let me check this out 👍